### DISCUSSION TITLE
Date of taken pictures

### DISCUSSION THREAD

**Discussion by Archonw** [Geektoolkit/Dynaframe3#77] [Ideas]
I would like to have the option to show the date where the picture was taken.
I use this program to show my family photos ( nearly 4000). And sometimes it is hard to remember when this pictures was taken.

**Comment by Geektoolkit** [comment]
Good feature, I'll add it to the list. I want the infobar to show date and hopefully more info in the future. We have some UI fixes coming from Rich that I want to get in first, and he's got an infobar fix, then we should be able to do stuff like this fairly quickly

**Reply by RichN001** [reply]
https://github.com/drewnoakes/metadata-extractor-dotnet

**Comment by msmcknight** [comment]
I'd like to 2nd this request.  Displaying the "EXIF Date/Time Original" would be great.  

Another nice option would be to display the slideshow in order of EXIF date... ie... show the pictures chronologically irrespective of filename or filedate.

**Reply by Geektoolkit** [reply]
Getting the time/date to show is fairly trivial, I'll work to get that in.

For exif date/time ordering, what's your thoughts/desire on if the data isn't there? Does it get filtered out? shown at the end ordered by filedate?

**Reply by msmcknight** [reply]
I think if the EXIF date is missing then just show the pictures at the beginning... ie assume the date is 0/0/0000 and go up from there.  Others may think differently so maybe some config options would be nice as future add-ons.

**Reply by Geektoolkit** [reply]
I'll likely start with that and then add options as requests come in. I think having anything for sorting via date will be handy for now :)

**Reply by Geektoolkit** [reply]
Good news, I'm able to pull the exif data out, and can see the date/time original values. I have one other thing to follow up on (someone asked about sorting by exif data) and when I get that answer I'll try to combine these into a feature release (ability to filter by exif, and sort by TimeOriginal)

**Reply by msmcknight** [reply]
Excellent!  Thanks for the update.
